### ISSUE TITLE
Needs further work to work properly

### ISSUE THREAD

**Issue by TomSeldon** [TomSeldon/intro-videos-app#1]
Misunderstood how Plex deals with intro videos. Expected it to load a video from a URL when in fact it appears to just do a regex on the given URL for the media ID (i.e. the URL is never actually requested).

As such, this solution isn't going to work. 

Keeping this repo around should Plex ever change how they handle loading pre-roll videos.

**Comment by davisengeler** [comment]
I'm glad I found this issue sooner than later. Haha. I also assumed this technique was going to do the trick. Thanks for your work anyway. You should update this if you come across a solution down the road.

**Comment by TomSeldon** [comment]
Thanks @davisengeler, shame this never worked out as it seemed like a cool idea! 

I'll take another look at this over the weekend, and if it still appears that it won't work, I'll remove the repo to avoid others mistakingly thinking this will work.

**Comment by davisengeler** [comment]
Nah, don't do that. Just add/update the readme to state the problem.
